### PR TITLE
oclint: new package

### PR DESCRIPTION
**Pull request by goxberry**

### CODE REVIEW

File: var/spack/repos/builtin/packages/oclint/package.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+# This is a template package file for Spack.  We've put "FIXME"
```
Comment (davydden) [inline var/spack/repos/builtin/packages/oclint/package.py]: fixme

Comment (goxberry) [reply var/spack/repos/builtin/packages/oclint/package.py]: Done.

**Comment by goxberry** [comment]
I think I'm done all of the minor tweaks, but if there's anything I missed re: style or anything else, let me know.

File: var/spack/repos/builtin/packages/oclint/package.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        bundle_script('-release', '-llvm-root={0}'.format(spec['llvm'].prefix))
+
+        # Copy install tree into the correct locations using the
+        # directory layout described in
+        cd(join_path('..', 'build'))
+        install_tree(join_path('oclint-release', 'include'), prefix.include)
+        install_tree(join_path('oclint-release', 'lib'), prefix.lib)
+        install_tree(join_path('oclint-release', 'bin'), prefix.bin)
+
+        return
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/oclint/package.py]: No need for a return statement here (if you wondered what was the issue that didn't appear in the review above).

**Comment by goxberry** [comment]
@alalazo Fixed.

**Pull request merged**